src/app/suspect/page.tsx

```
@@ -1,103 +1,94 @@
-                                { step.university } ({ step.years })
-                            </p>
+                    <div className="space-y-6">
+                        <div className="mb-8">
+                            <h2
+                                className="text-2xl font-bold text-primary mb-4 font-top-secret"
+                            >
+                                {t('suspect.softskills')}
+                            </h2>
+                            <ul className="space-y-2">
+                                {softSkills?.map(skill => (
+                                    <li className="font-top-secret">➤ {t(`${skill}`)}</li>
```

Biome (2.1.2)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)